django_mongodb/schema.py

```diff
@@ -29,18 +29,26 @@ def create_model(self, model):
 
     def _create_model_indexes(self, model):
         """
-        Create all indexes (field indexes, index_together, Meta.indexes) for
-        the specified model.
+        Create all indexes (field indexes & uniques, Meta.index_together,
+        Meta.unique_together, Meta.constraints, Meta.indexes) for the model.
         """
         if not model._meta.managed or model._meta.proxy or model._meta.swapped:
             return
-        # Field indexes
+        # Field indexes and uniques
         for field in model._meta.local_fields:
             if self._field_should_be_indexed(model, field):
                 self._add_field_index(model, field)
+            elif self._field_should_have_unique(field):
+                self._add_field_unique(model, field)
         # Meta.index_together (RemovedInDjango51Warning)
         for field_names in model._meta.index_together:
             self._add_composed_index(model, field_names)
+        # Meta.unique_together
+        if model._meta.unique_together:
+            self.alter_unique_together(model, [], model._meta.unique_together)
+        # Meta.constraints
+        for constraint in model._meta.constraints:
+            self.add_constraint(model, constraint)
```

Comment:
Does this addition also mean that this PR will support `BaseConstraints`?

Reply:
`BaseConstraint` is just a common base class. It isn't something that's used on a model's `Meta.constraints`.